src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/EmbeddingService.java

```diff
@@ -0,0 +1,15 @@
+package io.stargate.sgv2.jsonapi.service.embedding.operation;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public interface EmbeddingService {
+  int EXECUTOR_SERICE_THREADS = 10;
+  int MAX_RETRIES = 3;
+  ExecutorService executor =
+      Executors.newFixedThreadPool(
+          EXECUTOR_SERICE_THREADS, r -> new Thread(r, "EMBEDDING_SERVICE_REQUEST"));
```

Comment:
EXECUTOR_SERICE_THREADS and RETRIES are not used in implementation.
That is because we are still doing API call mocking? and wait for the api keys from the company.